internal/connect/status.go

```diff
@@ -107,3 +107,32 @@ func getStatusText(statuses map[string]Status) (string, error) {
 	}
 	return output.String(), nil
 }
+
+// SystemProducts returns sum of installed and activated products
+// Products from zypper have priority over products from
+// activations as they have summary field which is missing
+// in the latter.
+func SystemProducts() ([]Product, error) {
+	products, err := installedProducts()
+	if err != nil {
+		return products, err
+	}
```

Comment:
maybe `if !IsRegistered() { return products, nil }` here instead of the check+indent below?

Reply:
done